test/e2e/get-bridge-vlans-flags-el8.sh

```diff
@@ -7,5 +9,7 @@ vlans_out=/tmp/vlans.out
 
 ./kubevirtci/cluster-up/ssh.sh $node -- sudo bridge vlan show > $vlans_out
 
+dos2unix $vlans_out
```

Comment:
What is this?

Reply:
After updating kubevirtci it dumps non unix new line (\r\n thingy) we have to transform it so we are able to parse (centos8 iptools "bridge -j vlan show is not json complaing so we have to use this script)